### PR TITLE
Add clarification for case-sensitivity for LicenseRef-

### PR DESCRIPTION
**Pull request by vargenau**
See https://github.com/spdx/spdx-spec/issues/981

### CODE REVIEW

**Review by goneall — APPROVED** [review]
This resolves an inconsistency in the spec.  Note that the SPDX Java tools does not currently support this clarification, however.  There is an existing issue to track updating the SPDX Java tools.

**Review by lpanni — APPROVED** [review]

File: docs/annexes/SPDX-license-expressions.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ License identifiers (including license exception identifiers) used in SPDX docum
 
 However, please be aware that it is often important to match with the case of the canonical identifier on the [SPDX License List](https://spdx.org/licenses). This is because the canonical identifier's case is used in the URL of the license's or exception's entry on the List, and because the canonical identifier is translated to a URI in RDF documents.
 
+For license identifiers, only the variable part (after `LicenseRef-`) is case insensitive.
+
+So, `LicenseRef-Beerware-4.2` and `LicenseRef-beerware-4.2` are the same identifier, and `licenseref-beerware-4.2` is not a valid license identifier.
```
Comment (zvr) [inline docs/annexes/SPDX-license-expressions.md:61]:
```suggestion
This means, for example, that `LicenseRef-Name` and `LicenseRef-name` should be treated as the same identifier and considered to refer to the same license, while `licenseref-name` is not a valid license identifier.
```

**Review by zvr — CHANGES_REQUESTED** [review]
And I would also add this line to the previous paragraph (after "case insensitive.") instead of having it as a separate paragraph.

**Comment by vargenau** [comment]
@zvr 
Merging is blocked by DCO.
Should I do something?

**Comment by bact** [comment]
@vargenau  DCO requires you to signed-off the commit again (for every commits).

**Comment by vargenau** [comment]
> @vargenau DCO requires you to signed-off the commit again (for every commits).

Is it me or @zvr as he made a change to my commit?

**Comment by bact** [comment]
Anyone who actually make the commit.
(in this case, zvr only suggested the change but not actually commit the change himself).

Try this to rebase https://github.com/spdx/spdx-spec/pull/984/checks?check_run_id=27520863733
(not always successful, to my experience, as there can be some conflicts that can't be rebased. but still worth a try)

**Comment by vargenau** [comment]
> Anyone who actually make the commit. (in this case, zvr only suggested the change but not actually commit the change himself).
> 
> Try this to rebase https://github.com/spdx/spdx-spec/pull/984/checks?check_run_id=27520863733 (not always successful, to my experience, as there can be some conflicts that can't be rebased. but still worth a try)

Thank you @bact for your help. It's done. 

Please merge.

**Comment by goneall** [comment]
It looks like the requested change from @zvr has been made, going ahead and merging.

**Pull request merged**